src/main/cpp/src/bloom_filter.cu

```diff
@@ -31,6 +31,7 @@
 
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/exec_policy.hpp>
+#include <rmm/resource_ref.hpp>
```

Comment:
Including the header in `cu` file is redundant, since we already did that in the header of this source file (`bloom_filter.hpp`).

This is applied for all other source files.

Reply:
Done.